spec/features/notes_spec.rb

```diff
@@ -49,6 +49,19 @@
       sleep 0.5
       expect(find('#in_progress .story .notelist')).not_to have_content('Delete me please')
     end
+
+    it 'does not save an invalid note', js: true do
+      visit project_path(project)
+
+      within('#in_progress .story') do
+        find('.story-title').trigger('click')
+        fill_in 'note', with: nil
+        click_on 'Add note'
+      end
+
+      sleep 0.5
```

Comment:
We shouldn't need it, why is it necessary?

Reply:
When a note doesn't have a description it throws an error. I think we can prevent it in front-end disabling the `Add Note` button, what do you think @talyssonoc ?

Reply:
I think this is a good idea, could you check it @GabrielRMuller

Reply:
yea, no problem.